test/e2e/l7networkpolicy_test.go

```diff
@@ -333,3 +337,55 @@ func testL7NetworkPolicyTLS(t *testing.T, data *TestData) {
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "apis.google.com", false)
 	probeL7NetworkPolicyTLS(t, data, clientPodName, "www.facebook.com", true)
 }
+
+func testL7NetworkPolicyLogging(t *testing.T, data *TestData) {
+	l7LoggingNode := nodeName(0)
+	// L7 audit logging filename is defined in Suricata config in /etc/suricata/antrea.yaml as "eve-%%Y-%%m-%%d.json".
```

Comment:
don't we generate the Suricata config programmatically using code? We should probably add a link to that code instead

Reply:
Some config fields are handwritten, I pointed to the config in `reconciler.go` where filename is defined, is that what you meant?